### DISCUSSION TITLE
OpenAI API Key is not provided

### DISCUSSION THREAD

**Discussion by Floxxman** [logancyang/obsidian-copilot#560] [Q&A]
Why do i need an OpenAI API Key while using a local model for vaultQA?

I get an error while trying to index for my vault.
<img width="1438" alt="Screenshot copilot" src="https://github.com/logancyang/obsidian-copilot/assets/10404310/57999b70-f71a-4e12-a55b-75c5bed50ba3">

LM Studio is up and running. I can chat, but not vaultQA.

**Comment by logancyang** [comment]
What embedding model did you pick? Can you share a screenshot of your setting.

The example I posted was using Ollama because Ollama can spin up 2 servers at the same time, a chat model and an embedding model. Not sure if LM Studio can do that?
